
The staging driver rtlwifi/r8822be.ko.xz is being replaced by the new rtw88 driver.

Add a module alias to keep upgrade path.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- a/drivers/net/wireless/realtek/rtw88/rtw8822be.c
+++ b/drivers/net/wireless/realtek/rtw88/rtw8822be.c
@@ -28,3 +28,4 @@ module_pci_driver(rtw_8822be_driver);
 MODULE_AUTHOR("Realtek Corporation");
 MODULE_DESCRIPTION("Realtek 802.11ac wireless 8822be driver");
 MODULE_LICENSE("Dual BSD/GPL");
+MODULE_ALIAS("r8822be");
